### PR TITLE
chore: remove all remaining unused exports

### PR DESCRIPTION
**Pull request by vm**
Removes all remaining dead exports! Now we can add this as a condition on our build.

### CODE REVIEW

File: src/components/FeatureFlagModal/FeatureFlagModal.tsx [hunk 1]
```diff
@@ -209,6 +210,14 @@ export default function FeatureFlagModal() {
           label="Enables JSON-RPC tracing"
         />
       </FeatureFlagGroup>
+      <FeatureFlagGroup name="Tokens">
```
Comment (zzmp) [inline src/components/FeatureFlagModal/FeatureFlagModal.tsx:213]: Should this be in a separate PR?

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/components/FeatureFlagModal/FeatureFlagModal.tsx (hunk 1 above)
Comment (vm) [reply src/components/FeatureFlagModal/FeatureFlagModal.tsx:213]: i can do in sep pr yes, but i did it so we had no more unused things

Comment (vm) [reply src/components/FeatureFlagModal/FeatureFlagModal.tsx:213]: https://github.com/Uniswap/interface/pull/5511

**Review by zzmp — APPROVED** [review]
just merge it after #5511
